### ISSUE TITLE
собственные звуки на Alert

### ISSUE THREAD

**Issue by RoketAnny** [SBProX/SBProX-Dev#605]
Можно вернуть возможность использовать свои звуки в алертах добавляя их в папку alerts как в прошлой версии. 

В текущей версии, при добавлении файла,  в окне выбора звукового оповещения добавленные файлы не отражаются :(

**Comment by r0b4ik** [comment]
Да, очень странно, т.к. выглядит, что архитектура не поменялась, та же папка, те же файлы, что и в старой версии, но своих файлов в списке нет. Возможно вы специально заблоктровали эту функцию, т.к. были жалобы, что свои файлы пропадают после обновления. Но лучше каждый раз их перезаписывать, чем вообще не иметь такой возможности
